ViewModel/Adminhtml/Common.php

```diff
@@ -217,6 +229,39 @@ public function isClickAnalyticsTurnedOnInAdmin()
         return $this->configHelper->isClickConversionAnalyticsEnabled();
     }
 
+    public function isMysqlUsed()
+    {
+        return $this->configHelper->getBackendSearchEngine() === 'mysql';
+    }
```

Comment:
\Magento\Search\Model\EngineResolver::CATALOG_SEARCH_MYSQL_ENGINE const can be used here instead of matching against string.